client/src/App.css

```diff
@@ -99,3 +99,207 @@ button{
   }
 
 }
+
```

Comment:
These style changes should not be on the App level. 

A style.css file should exist in the client/src/components/messageRoom folder. In the index.js for the message room have a "import './style.css'. 

This will prevent merge issues from occurring concerning the App.css file.